frontend/utilities/permissions/permissions.ts

```diff
@@ -102,6 +109,15 @@ const isOnlyObserver = (user: IUser): boolean => {
   return false;
 };
 
+const isObserverPlus = (user: IUser, teamId: number | null): boolean => {
```

Comment:
Maybe call this `isGlobalObserverPlus` to clearly differentiate between this and the `team`-level option?

Reply:
Is there a reason we want to differentiate between Global and Team? This role works for both global (line 113) and team (line 117).